frontend/coprs_frontend/coprs/logic/batches_logic.py

```diff
@@ -29,9 +30,9 @@ def _lock_table(table):
     # (while reading is not blocked) until the end of the transaction
     # (commit / rollback)
     with db.engine.connect() as connection:
-        connection.execute("LOCK TABLE {} IN EXCLUSIVE MODE;".format(
-            sa_quote(db.engine, table),
-        ))
+        connection.execute(text("LOCK TABLE {} IN EXCLUSIVE MODE;".format(
```

Comment:
so if `text` is needed for raw SQL, then what about these:
- https://github.com/fedora-copr/copr/blob/main/frontend/coprs_frontend/coprs/logic/batches_logic.py#L32
- https://github.com/fedora-copr/copr/blob/main/frontend/coprs_frontend/coprs/logic/builds_logic.py#L175
- https://github.com/fedora-copr/copr/blob/main/frontend/coprs_frontend/coprs/logic/builds_logic.py#L196

Reply:
That is a good point. The first one is definitely a problem. Thank you for catching that.
The other two are running `connection.execute(query, params)` on a `query` which is a `text`.

Reply:
Let's have a new PR for the other problem.